nhost/service/manager.go

```diff
@@ -19,6 +19,10 @@ import (
 	"time"
 )
 
+const (
+	slowStartWaitMsg = "It takes more than usual to start the nhost project. Most likely because CLI needs to pull js dependencies inside a container. Please wait..."
```

Comment:
"It takes more than usual to start the nhost project. Most likely because CLI needs to pull js dependencies inside a container. Please wait..."

=> 

"It takes longer than usual to start your Nhost project. Most likely because of the need to install your npm dependencies. Please wait."